src/IO/HDF5/HDF5IOHandler.cpp

```diff
@@ -81,6 +84,40 @@ HDF5IOHandlerImpl::HDF5IOHandlerImpl(AbstractIOHandler* handler)
     H5Tinsert(m_H5T_CDOUBLE, "i", sizeof(double), H5T_NATIVE_DOUBLE);
     H5Tinsert(m_H5T_CLONG_DOUBLE, "r", 0, H5T_NATIVE_LDOUBLE);
     H5Tinsert(m_H5T_CLONG_DOUBLE, "i", sizeof(long double), H5T_NATIVE_LDOUBLE);
+
+    m_chunks = auxiliary::getEnvString( "OPENPMD_HDF5_CHUNKS", "auto" );
+    // JSON option can overwrite env option:
+    if( config.contains( "hdf5" ) )
+    {
+        m_config = std::move( config[ "hdf5" ] );
+
+        // check for global dataset configs
+        if( m_config.json().contains( "dataset" ) )
+        {
+            auto datasetConfig = m_config[ "dataset" ];
+            if( datasetConfig.json().contains( "chunks" ) )
+            {
+                m_chunks = datasetConfig.json()[ "chunks" ];
```

Comment:
```suggestion
                m_chunks = datasetConfig[ "chunks" ].json().get< std::string >();
```
Calling `operator[]()` on the `TracingJSON` object will trace the key. I think it's good to be explicit with the type that you want, I already ran in to trouble because I wasn't in the past.